### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by mgoodhand**
Discourage use of PyPI until it's been updated to use the new version (#108).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,6 +30,8 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed. It doesn't work with Python 3 yet, see `#40 <https://github.com/linkcheck/linkchecker/pull/40>`_ for details.
 
+`pip install linkchecker` should NOT be used for now, as it will install the old version of linkchecker. See `#4.
```
Comment (mgedmin) [inline README.rst]: The (rather silly) ReStructuredText syntax for hyperlinks is

    `link text <URL>`_

Can you please modify this to say

     See `#4 <https://..>`_.

?

Comment (mgedmin) [reply README.rst]: Oh, and I nearly didn't notice -- the ReST syntax for monospace text is double backticks, so

    ``pip install linkchecker``

Comment (mgoodhand) [reply README.rst]: Thanks for the feedback. ReST clearly not my strong point (who'd've guessed README could be so complicated). Will fix ...

**Comment by mgoodhand** [comment]
I think I've fixed the markup now. Sorry for the faff.

**Pull request merged**